packages/core/admin/admin/src/content-manager/components/SingleTypeFormWrapper/index.js

```diff
@@ -50,6 +50,50 @@ const SingleTypeFormWrapper = ({ allLayoutData, children, slug }) => {
   const { componentsDataStructure, contentTypeDataStructure, data, isLoading, status } =
     useSelector(selectCrudReducer);
 
+  useQuery(
+    [slug, params],
```

Comment:
```suggestion
    ['content-manager', slug, params],
```

Reply:
Actually this made me realize I forgot to remove useQuery when I reverted back to the original useEffect that was in place 🤦‍♂️ . I plan on opening a PR to migrate to useQuery for collection-types and single-types after this one is merged. So I'll apply this feedback there.